File: User_management/logic/user_handlers/service_health.go
package userhandlers

import "github.com/gofiber/fiber/v2"

// UserManagerServiceHealthCheck godoc
// @Summary Check the health of the user management service
// @Description Get the health status of the user management service
// @Tags health
// @Produce json
// @Success 200 {object} map[string]string
// @Router /health [get]
func UserManagerServiceHealthCheck(c *fiber.Ctx) error {
	return c.Status(fiber.StatusOK).JSON(fiber.Map{"Health check user management service":"Ok"})
}

File: task_management/logic/handlers/service_health.js
/**
 * @swagger
 * /health:
 *   get:
 *     summary: Check the health status of the service
 *     tags: [Health]
 *     responses:
 *       200:
 *         description: Service is running
 *         content:
 *           application/json:
 *             schema:
 *               type: object
 *               properties:
 *                 status:
 *                   type: string
 *                   example: 'Task manager service is running Ok!'
 *       500:
 *         description: Internal server error
 */
async function checkHealth(req, res) {
    res.json({ status: 'Task manager service is running Ok!' });
  }
  
  module.exports = {
    checkHealth,
  };
  